hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -72,6 +72,31 @@
             FIFO was full.
             '''
     }
+    { name: "link_in_err",
+      desc: '''
+            Raised if a rollback occured in an IN endpoint.
```

Comment:
Might want to find a more programmer friendly term than "rollback"? 
Maybe "Raised if a packet to an IN endpoint started to be received but was then dropped due to an error"

Reply:
Yes, that's clearer. New version:

Raised if a packet to an IN endpoint started to be received but was
then dropped due to an error. After transmitting the IN payload,
the USB device expects a valid ACK handshake packet. This error is
raised if either the packet or CRC is invalid or a different token
was received.